runtime/compiler/codegen/J9TreeEvaluator.cpp

```diff
@@ -499,32 +499,25 @@ bool J9::TreeEvaluator::getIndirectWrtbarValueNode(TR::CodeGenerator *cg, TR::No
          translatedNode = translatedNode->getFirstChild();
          }
 
-      if (translatedNode->getOpCode().isSub() ||
-           TR::Compiler->vm.heapBaseAddress() == 0 || sourceChild->isNull()) /* i.e. usingLowMemHeap */
+      usingCompressedPointers = true;
```

Comment:
Can we fold uses of this as true?

Reply:
I don't think there's any benefit to folding this one. `usingCompressedPointers` is the return value for this routine. It's initialized as `false` and only changed to `true` if the condition on line 466 above is met:
```
   if (cg->comp()->useCompressedPointers() && (node->getSymbolReference()->getSymbol()->getDataType() == TR::Address) &&
         (node->getSecondChild()->getDataType() != TR::Address))
```